docs/spfx/library-component-tutorial.md

```diff
@@ -106,7 +115,7 @@ localization_priority: Priority
     gulp serve
     ```
 
-## How to deploy and consume a 3rd party SPFx library from tenant app catalog
+## How to deploy and consume a 3rd party SPFx library from the tenant app catalog
 
 1. Navigate to the ```corporate-library``` root folder and bundle and package the solution:
```

Comment:
Inline code references should be surrounded with a single backtick, not triple. Triple backticks are used to fence code blocks, not inline code.

Reply:
Again bad habbit.